modules/nw-operator-cr.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * networking/cluster-network-operator.adoc
 // * installing/installing_aws/installing-aws-network-customizations.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
 
 ifeval::["{context}" == "installing-aws-network-customizations"]
```

Comment:
Do you need to update the if statements for Azure?

Reply:
Reworked conditionals.